docs/api-reference.md

```diff
@@ -265,6 +265,16 @@ function MyComponent:didMount()
 end
 ```
 
+Since lua tables interpret a `nil` value as absence, removing a value from state is not possible by setting its value to `nil`. If a value needs to be removed from state, it can be set to `Roact.None` when calling `setState`:
```

Comment:
Can we tweak this message a little bit?

I want to emphasize that it's *literally impossible* to have a value of `nil` in a table. Having a value of `nil` is exactly the same as not having the value at all!

Reply:
How do you feel about:

Note that removing a value from state is not possible by setting its value to `nil` because, in lua tables, `nil` values mean the same thing as no value at all. If a value needs to be removed from state, it can be set to `Roact.None` when calling `setState`, which will ensure that the resulting state no longer contains it:

Reply:
Make sure you capitalize Lua, and then that would be fine I think.